core/openapi/schemas/contact.yaml

```diff
@@ -3,28 +3,31 @@ type: object
 description: |-
   Identification of, and means of communication with, person responsible
   for the resource.
-required:
-  - name
+anyOf:
+  - allOf:
```

Comment:
You don't need the allOf in both branches :-)

Reply:
Confirmed this continues to work as expected.  PR updated.